lib/config.js

```diff
@@ -212,11 +235,7 @@ function parseED25519 (prefix) {
 
   let keyPair
   if (!ed25519.secret) {
-    if (process.env.NODE_ENV === 'production') {
-      throw new ServerError('No ' + (prefix ? prefix.toUpperCase() + '_' : '') +
-        'ED25519_SECRET_KEY provided.')
-    }
-    const seed = crypto.randomBytes(32)
+    const seed = generateSecret(prefix, 'ed25519')
```

Comment:
Or, the map could initially be empty, then add an entry keyed by `name` any time `generateSecret` is called. If `generateSecret` is called with a repeated name, it could throw.